test/unit/proj_context_test.cpp

```diff
@@ -189,4 +192,20 @@ TEST(proj_context, proj_context_set_ca_bundle_path) {
     proj_context_destroy(ctx);
 }
 
+// ---------------------------------------------------------------------------
+
+TEST(proj_context, proj_context_clone) {
+    std::string dbpath("/tmp/dummmy/path/proj.db");
+    PJ_CONTEXT *default_ctx = pj_get_default_ctx();
+    projCppContext* original_cpp_context = default_ctx->cpp_context;
+    default_ctx->cpp_context = new projCppContext(default_ctx, dbpath.c_str(), {});
```

Comment:
you cannot do this. This is an internal symbol of the lib, not accessible by the tests. 
You could possibly avoid calling it directly by using proj_context_set_database_path() instead

Reply:
I thought about that. I tried this route as that method looks like it attempts to connect to the database and reverts it fails. Sounds like I will need to go that route. Thanks for the explanation for why that failed 👍